vector-im/element-android
# Code Style Change - Method Parameters on new line from parentheses

## Type of change


## Content

Applies the codes style change to move (multiline) method parameters on a new line from both the opening and closing parentheses

## Motivation and context

Closes https://github.com/vector-im/element-android/issues/5771

See also https://github.com/vector-im/element-android/pull/5644

I tried to isolate this change to the mentioned issue as much as possible, but it seems the autoformatter wanted to correct the inconsistencies that weren't on the current code style

I tested conflicts for this by merging this branch into the [Tchap fork](https://github.com/tchapgouv/tchap-android) and this seemed to introduce only a few more easy-to-resolve conflicts than there would be when merging with develop

I figured if the forks perform the same code style change strategy as undertaken to make the changes on this PR, they can encounter even less conflicts. I will document this strategy later, but in a nutshell, that strategy is:

1. Strip down the `.editorconfig` file to only the changes being made

```
[{*.gradle.kts,*.kt,*.kts,*.main.kts}]
ij_kotlin_method_parameters_new_line_after_left_paren = true
ij_kotlin_method_parameters_right_paren_on_new_line = true
```

2. Reformat the modules with file mask `.kt`

## Screenshots / GIFs

N/A

## Tests



- Checkout Tchap fork
- Add the original element repo as upstream and fetch all the branches
- Merge this branch with Tchap develop

You will see that there are a lot of conflicts, but most of these conflicts come from changes that are on our repo's develop and not the changes on this branch specifically. If you repeat the steps above but instead merge upstream/develop into origin/develop, you will see that most of those conflicts are still present and this branch adds only few.

## Tested devices

- OS version(s):

## Checklist
